### ISSUE TITLE
Separate upcoming events and previous events on the news page

### ISSUE THREAD

**Issue by WhiteFox08** [fparma/fparma-web#19]
![21wd39](https://cloud.githubusercontent.com/assets/2702850/13730156/a0e3c7e6-e915-11e5-8dbc-b1a71f7509b1.png)

Something like a double pane would fix this??

**Comment by profan** [comment]
Could just have a "UPCOMING" title above the list of upcoming ones and
"PAST" above the section for the ones in the past, would be simple and
look decent?

**Comment by Cuel** [comment]
The original idea was that admins could write news aswell, not just that it would be spammed with events.

This suggestion would make it just the like the /events page, and then I don't see a reason to have both, pretty redundant

I see different solutions
1) Remove the current auto-post news entirely and make it so admins can write news instead, e.g modpack updates or other stuff.
2) Render the event news with a [horizontal label](http://semantic-ui.com/elements/label.html#horizontal) to show that it's upcoming / in the past. E.g it would look like a tag.

**Comment by profan** [comment]
Actually, couldn't you have upcoming events on the front-page as well
above the News pane? Maybe showing past events isn't necessary on the
front-page though, but upcoming events is one of those things I'd think
is what most people are looking for anyways.

Not making every event a news-item would be good, just having them
separate would make things clearer I think.

**Comment by Cuel** [comment]
Yeah a small panel for upcoming events is a good idea. It doen't have to be anymore than the link and date
